results/customer-portal/installer-platform.detailed-description.html

```diff
@@ -0,0 +1,12 @@
+The Red Hat JBoss Developer Studio Platform Installer scans for the products listed below using your PATH environment variable to determine if they're already installed. If not found, or an older version is installed, the following supported versions will then be installed for you by the installer:
```

Comment:
I borrowed that text from https://docs.google.com/document/d/19UHqLg1Az0N23FM1d3PijSZt6rDjAC9QjhidtbJErPE/edit# so... we can simplify it further.